src/main/java/org/jenkins/tools/test/PluginCompatTesterCli.java

```diff
@@ -54,85 +54,157 @@ public class PluginCompatTesterCli {
-            if (options.isPrintHelp()) {
-                jcommander.usage();
-                return;
-            }
-        } catch (ParameterException e) {
-            System.err.println(e.getMessage());
-            jcommander.usage();
-            System.exit(1);
-        }
+    @CommandLine.Option(
+            names = {"-w", "--war"},
+            required = true,
+            description = "Path to the WAR file to be used by the PCT.")
+    private File war;
+
+    @CommandLine.Option(
+            names = "--working-dir",
+            required = true,
+            description = "Working directory where plugin sources will be checked out")
+    private File workingDir;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = "--include-plugins",
+            split = ",",
+            arity = "1",
+            paramLabel = "plugin",
+            description =
+                    "Comma-separated list of plugin artifact IDs to test. If not set, every plugin in the WAR will be tested.")
+    private List<String> includePlugins;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = "--exclude-plugins",
+            split = ",",
+            arity = "1",
+            paramLabel = "plugin",
+            description =
+                    "Comma-separated list of plugin artifact IDs to skip. If not set, only the plugins specified by --plugins will be tested (or all plugins otherwise).")
+    private List<String> excludePlugins;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = "--exclude-hooks",
+            split = ",",
+            arity = "1",
+            paramLabel = "hook",
+            description =
+                    "Comma-separated list of hooks to skip. If not set, all hooks will be executed.")
+    private List<String> excludeHooks;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = "--fallback-github-organization",
+            description =
+                    "Include an alternative organization to use as a fallback to download the plugin. It is useful for using your own fork releases for a specific plugin if the version is not found in the official repository. If set, The PCT will try to use the fallback if a plugin tag is not found at the regular URL.")
+    private String fallbackGitHubOrganization;
+
+    @CheckForNull
+    @CommandLine.Option(names = "--mvn", description = "The path to the Maven executable.")
+    private File externalMaven;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = "--m2-settings",
+            description = "Settings file to use when executing Maven.")
+    private File m2Settings;
+
+    @CheckForNull
+    @CommandLine.Option(
+            names = {"-D", "--define"},
+            description =
+                    "Define a user property to be passed to Maven when running tests. Note that these will NOT be passed to Maven during compilation.")
+    private Map<String, String> mavenProperties;
```

Comment:
Note the critical change in semantics: these are no longer passed to Maven during compilation.